brille_spinwaves/get_brille_toolbox.py

```diff
@@ -0,0 +1,24 @@
+import requests, json
+
+def get_release_tbx(url):
+    response = requests.get(url)
+    if response.status_code != 200:
+        raise RuntimeError('Could not query Github if release exists')
+    response = json.loads(response.text)
+    # Latest release is always the first entry
+    tbx = [v for v in response[0]['assets'] if v['name'].endswith('mltbx')][0]
+    return tbx
+
+def download_github(gh_response):
+    headers = {"Accept":"application/octet-stream"}
+    filename = gh_response['name']
+    url = gh_response['url']
+    with requests.get(url, stream=True, headers=headers) as r:
+        with open(filename, 'wb') as f:
+            for chunk in r.iter_content(chunk_size=8192):
+                f.write(chunk)
+    return filename
+
+if __name__ == '__main__':
+    tbx = get_release_tbx('https://api.github.com/repos/brille/brillem/releases')
```

Comment:
Does this include brille? (since it says brillem). Where is the actual download done in Jenkins (as opposed to the name==main section) to call these functions?

Reply:
Ah, `brillem` is the Matlab part of the code which is distributed as a toolbox, which this script gets... I should probably write it in Matlab in the mfile but I already had the Python code to copy and paste... 

The Python `brille` module is installed in the [`Jenkinsfile`](https://github.com/pace-neutrons/pace-integration/blob/brille_plus_spinw/brille_spinwaves/Jenkinsfile#L113) with `pip`.